.github/workflows/cd.yml

```diff
@@ -312,7 +312,7 @@ jobs:
 
   ci:
     name: CI
-    uses: grafana/plugin-ci-workflows/.github/workflows/ci.yml@main # zizmor: ignore[unpinned-uses]
+    uses: grafana/plugin-ci-workflows/.github/workflows/ci.yml@ci-cd-workflows/v1.1.2 # release-please-version
```

Comment:
any specific reason behind this comment - `#releas-please-version` or it's just there to remind where its coming from?